extensions/objects/templates/set-of-normalized-string-editor.component.spec.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for the list of normalized string editor.
+ */
+
+
+describe('SetOfNormalizedStringEditor', function() {
+  var ctrl = null;
+
+  beforeEach(angular.mock.module('oppia'));
+  beforeEach(angular.mock.inject(function($componentController) {
+    ctrl = $componentController('setOfNormalizedStringEditor');
+  }));
+
+  it('should initialize the value with an empty array', function() {
```

Comment:
Is it possible to test the initial state being bound via an instantiation of the directive using the value parameter? What about actually interacting with the list to make sure it properly updates the value list? In general, we probably shouldn't rely on the implementation details (schema editor) being thoroughly tested--if the implementation of list editor ever changed, it would essentially go untested without tests existing to verify its contract.

Reply:
Is it okay if this becomes a TODO issue? I don't have a lot of time the next few weeks to add this.

Reply:
I think that's fine since it seems like this isn't a pattern in other similar editors (but probably should be).

Reply:
Yup -- please could you file it and add a TODO comment here? Thanks!

Reply:
Added a comment, and filed an issue here https://github.com/oppia/oppia/issues/11014